core/app/app.php

```diff
@@ -163,12 +171,23 @@ private function enqueue_assets() {
 			true
 		);
 
+		wp_register_script(
+			'select2',
+			ELEMENTOR_ASSETS_URL . 'lib/e-select2/js/e-select2.full.js',
```

Comment:
Add a minified version via `$suffix = ( defined( 'SCRIPT_DEBUG' ) && SCRIPT_DEBUG || defined( 'ELEMENTOR_TESTS' ) && ELEMENTOR_TESTS ) ? '' : '.min';`

Reply:
You can use $this->get_js_assets_url() for this purpose